beams/behavior_tree/ActionNode.py

```diff
@@ -1,33 +1,38 @@
 import atexit
 import os
 from multiprocessing import Event, Lock
+from typing import Callable, Any, Optional, Union
 
 import py_trees
 from epics.multiproc import CAProcess
 
+from beams.behavior_tree.ActionWorker import ActionWorker
 from beams.behavior_tree.VolatileStatus import VolatileStatus
 
 
 class ActionNode(py_trees.behaviour.Behaviour):
     def __init__(
         self,
-        name,
-        work_func,
-        completion_condition,
+        name: str,
+        work_func: Callable[[Any], None],
+        completion_condition: Callable[[Any], bool],
         work_gate=Event(),
         work_lock=Lock(),
         **kwargs,
     ):  # TODO: can add failure condition argument...
         super(ActionNode, self).__init__(name)
```

Comment:
Nitpick: this one of the old py2 syntaxes for super, it still works but it's wordier than the new way. I realize this is not part of the diff and that it matches the docs example but it caught my eye.
```suggestion
        super().__init__(name)
```